.github/workflows/e2e.yml

```diff
@@ -168,6 +168,7 @@ jobs:
       make-target: ${{ matrix.make-target }}
       runs-on: ${{ matrix.runs-on}}
       run: ${{ matrix.run }}
+      timeout-minutes: "${{ matrix.timeout-minutes || 25 }}"
```

Comment:
We have to set the default value on this side too since setting an empty string will overwrite the default value set in resusable-e2e.yml